src/module/item/ability/document.ts

```diff
@@ -50,6 +51,16 @@ class AbilityItemPF2e<TParent extends ActorPF2e | null = ActorPF2e | null> exten
         }
     }
 
+    override prepareActorData(): void {
+        const actor = this.actor;
+
+        if (actor?.isOfType("familiar") && this.system.category === "familiar") {
+            const prefix = "familiar:ability";
+            const slug = this.slug ?? sluggify(this.name);
+            actor.rollOptions.all[`${prefix}:${slug}`] = true;
```

Comment:
```suggestion
            const slug = this.slug ?? sluggify(this.name);
            actor.rollOptions.all[`ability:${slug}`] = true;
```

I personally don't know if we should include self: or not (perhaps @stwlam can answer that) but we definitely shouldn't include familiar: in actor.rollOptions.

Reply:
It probably should include `self:` since the master may need to query it.